extern/detray/CMakeLists.txt

```diff
@@ -13,13 +13,13 @@ message( STATUS "Building Detray as part of the TRACCC project" )
 
 # Declare where to get Detray from.
 set( TRACCC_DETRAY_SOURCE
-"URL;https://github.com/acts-project/detray/archive/refs/tags/v0.87.0.tar.gz;URL_MD5;0e30107edebfd41fbb3fc1974f665ea9"
+"GIT_REPOSITORY;https://github.com/niermann999/detray;GIT_TAG;ref-algebra-detail"
```

Comment:
Typo?

Reply:
Not really. I would like to include that branch in the new detray version

Reply:
So it's a draft PR, right...? Because of course we won't merge in this exact code. :thinking: Even a specific hash of your fork would be very iffy. But a branch name is of course absolutely a no-go.

Reply:
Sorry, I unmarked the draft, because it would otherwise not trigger the CUDA CI, which is what I am trying to fix on this detray branch